webpack/assets/javascripts/react_app/components/FactCharts/FactChartActions.js

```diff
@@ -1,26 +1,18 @@
 import {
-  FACT_CHART,
   FACT_CHART_MODAL_OPEN,
   FACT_CHART_MODAL_CLOSE,
 } from './FactChartConstants';
 import { get } from '../../redux/API';
 
-export const getChartData = (url, id) =>
-  get({
-    key: FACT_CHART,
-    url,
-    payload: { id },
-  });
-
-export const showModal = (id, title) => {
-  const showModalAction = {
+export const openModal = ({ key, url, id, title }) => dispatch => {
```

Comment:
It is confusing, I thought the `key` is something related to `modals` while it related to `API`.

Would it make sense calling it `apiKey`?
Or, don't pass the `key` and calculate it with the `id` you received?

Reply:
I understand your point, url and key doesn't seem related to open modal,
I prefer the key to be something which is declared once in one place.
Maybe we should revert to dispatch two actions in the index file as it was in the earlier commit?

Reply:
Why about?
```js
export const openModal = ({ id, title, apiKey, apiUrl }) => dispatch => {
  dispatch(get({ key: apiKey, url: apiUrl }));
};
```

IMO it looks clear enough